packages/rocketchat-importer-csv/server.js

```diff
@@ -254,7 +298,7 @@ Importer.CSV = class ImporterCSV extends Importer.Base {
 							const creator = this.getUserFromUsername(msg.username);
 							if (creator) {
 								const msgObj = {
-									_id: `csv-${ csvChannel.id }-${ msg.ts }`,
+									_id: `csv-${ csvChannel.id }-${ roomCounter }-${ msg.ts }`,
```

Comment:
@graywolf336 I tried to change the code so the IDs won't have an extra counter unless needed, so my changes won't break old re-import retries and noticed something odd.
It does work, but after the import the msgs counter in the admin rooms list double up. No duplicates in the room view that I noticed...
The funny thing, this happens with the old import code and IDs as well - duplicate messages counts, but unique when viewed.
Does this make sense? If so, I'll update the pull request with the safer code.